k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-configmap/templates/config-template.yaml

```diff
@@ -5,15 +5,15 @@ cortx:
         - tcp://<<.Values.cortx.external.kafka.endpoints>>:9092
       admin: admin
       secret: kafka_admin_secret
-    openldap:
-      endpoints:
-        - ldap://<<.Values.cortx.external.openldap.endpoints>>:389
-        - ssl://<<.Values.cortx.external.openldap.endpoints>>:636
-      servers:
-        <<.Values.cortx.external.openldap.servers>>
-      admin: admin
-      secret: openldap_admin_secret
-      base_dn: dc=seagate,dc=com
+    openldap:                                                          #Deleted
```

Comment:
Confirm from @kailashyadav11 if csm is getting deployed without openldap so we can remove those entires as pat of kailash ticket.

Reply:
@tanujashinde0405  Lets wait. As per protocol, entries goto deprecated first.

Reply:
Okay sure, We had a ticket to remove this entry: https://jts.seagate.com/browse/EOS-28037, But we can mark them as deprecated also.

Reply:
Fixed, changed to deprecated